charts/sonarqube/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # SonarQube Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [5.4.1]
+* Fix issue with additional network policy
```

Comment:
Can you please be more explicit about the issue that we fix? I'd suggest "Fix the right-dash curly brace issue with the additional network policy parameter"

Reply:
Thank you for spotting this as well, I didn't pay attention on it. Should be good now. :)

Reply:
@mydoomfr, I think you solved this conversation referring to the dce chart issue :) can you please fix the changelog entry?

Reply:
You're right, i was tired... Should be good now 😉